ansible/roles/test/tasks/crm.yml

```diff
@@ -4,7 +4,7 @@
      when: (testbed_type is not defined)
 
    - fail: msg="Invalid testbed_type value '{{testbed_type}}'"
-     when: testbed_type not in ['t1', 't1-lag', 't0', 't0-56', 't0-64', 't0-116']
+     when: testbed_type not in ['t0-8', 't1', 't1-lag', 't0', 't0-56', 't0-64', 't0-116']
```

Comment:
We’ll want to support this running on t1-8 also, since that’s the other one we have available.